mypy/fastparse.py

```diff
@@ -190,19 +192,27 @@ def as_block(self, stmts: List[ast35.stmt], lineno: int) -> Block:
 
     def fix_function_overloads(self, stmts: List[Statement]) -> List[Statement]:
         ret = []  # type: List[Statement]
-        current_overload = []
+        current_overload = []  # type: List[Decorator]
         current_overload_name = None
         # mypy doesn't actually check that the decorator is literally @overload
```

Comment:
Maybe update this comment a bit? It's still technically correct but less than before...